index.html

```diff
@@ -1,56 +1,188 @@
 <!DOCTYPE html>
 <html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<!-- Meta Descriptions -->
+		<meta
+			name="title"
+			content="Design for Everyone: A guide to inclusive web design"
+		/>
 
-<head>
-    <meta charset="utf-8">
-    <title>Placeholder</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
-    <link rel="stylesheet" href="styles/placeholder.css">
-</head>
-
-<body>
-    <div class="grid-container"><!-- BEGIN GRID CONTAINER  -->
-        <header>
-            <h1>Header Goes Here</h1>
-        </header>
-
-        <nav><h2>Menu Goes Here</h2>
-            <ul>
-                <li>Home</li>
-                <li>Catiline</li>
-                <li>Deiotarus</li>
-                <li>Murena</li>
-                <li>Caelius</li>
-                <li>Pompey</li>
-                <li>House</li>
-                <li>Provinces</li>
-                <li>Philippic</li>
-            </ul>
-        </nav>
-
-        <main>
-            <h2>Main Goes Here</h2>Cicero has been traditionally considered the master of Latin prose, with Quintilian
-            declaring Cicero was "not the name of a man, but of eloquence itself." He is credited with transforming
-            Latin from a modest utilitarian language into a versatile literary medium capable of expressing abstract and
-            complicated thoughts with clarity.
-			
-        </main>
-
-        <aside>
-            <h2>Aside Goes Here</h2><br>Quintilian declaring Cicero was "not the name of a man, but of eloquence
-            itself." He is credited with transforming Latin from a modest utilitarian language into a versatile literary
-            medium capable of expressing abstract and complicated
-        </aside>
-
-        <footer>
-            <h2>Footer Goes Here</h2>During the chaotic latter half of the first century BC, marked by civil wars and
-            the dictatorship of Gaius Julius Caesar, Cicero championed a return to the traditional republican
-            government.
-            <p>Thanks to Dr. William Helling of IUPUI and w3schools.com (https://www.w3schools.com/css/css_grid.asp) -- Kasper Valtakari</p>
-        </footer>
-
-    </div><!-- END GRID CONTAINER  -->
-
-</body>
-
-</html>
\ No newline at end of file
+		<meta
+			name="description"
+			content="Helping developers make their sites accessible for all."
+		/>
+
+		<meta
+			name="keywords"
+			content="inclusive, inclusive design, design, colorblind, screen reader, accessible, accessibility"
+		/>
+
+		<!-- Fonts: ROBOTO 300, 400, 500, 700 -->
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
+			rel="stylesheet"
+		/>

```

Comment:
We can remove this from each page to save some network bandwidth and use system fonts instead by adding the following code to the CSS: 

```
* {
        font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Ubuntu', 'Droid Sans', 'Helvetica Neue', sans-serif;
}
```